### ISSUE TITLE
Export error with new plugin 1.38.0

### ISSUE THREAD

**Issue by OnlyAnEngineer** [MyPureCloud/terraform-provider-genesyscloud#1040]
I've just attempted to do a full export with the new plugin version 1.38.0 with the below parameters. Note that it completes successfully when reverting back plugin 1.35.0

```
terraform {
  required_version = "~> 1.7.5"
  required_providers {
    genesyscloud = {
      source  = "mypurecloud/genesyscloud"
      version = "~> 1.38.0"
    }
  }
}

provider "genesyscloud" {
  # OAuth (client secret)
  oauthclient_id     = var.client_id
  oauthclient_secret = var.client_secret
  aws_region         = var.aws_region
}

resource "genesyscloud_tf_export" "export_prod" {
  directory                = "./export_prod"
  include_state_file       = true
  export_as_hcl            = false
  log_permission_errors    = true
  split_files_by_resource  = true
  enable_dependency_resolution = true
}
```

After a few mins of running it fails with error:

│ Error: Plugin did not respond
│
│   with genesyscloud_tf_export.export,
│   on prod.tf line 18, in resource "genesyscloud_tf_export" "export_prod":
│   18: resource "genesyscloud_tf_export" "export_prod" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.

Stack trace from the terraform-provider-genesyscloud_v1.38.0.exe plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x10b6c74]

goroutine 51159 [running]:
terraform-provider-genesyscloud/genesyscloud/outbound_settings.readOutboundSettings.func1()
        terraform-provider-genesyscloud/genesyscloud/outbound_settings/resource_genesyscloud_outbound_settings.go:76 +0x3b4
github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry.RetryContext.func1()
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.33.0/helper/retry/wait.go:30 +0x56
github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry.(*StateChangeConf).WaitForStateContext.func1()
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.33.0/helper/retry/state.go:113 +0x1c4
created by github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry.(*StateChangeConf).WaitForStateContext
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.33.0/helper/retry/state.go:86 +0x1d8

Error: The terraform-provider-genesyscloud_v1.38.0.exe plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

**Comment by dginty4** [comment]
Hi, thanks for point this out, we have a fixed merged and will do a point release on the latest version to have this resolved

**Comment by dginty4** [comment]
@OnlyAnEngineer This issue should be fixed in the latest version of the provider, v1.38.1
